sechub-doc/src/docs/asciidoc/documents/shared/configuration/sechub_config.adoc

```diff
@@ -300,6 +300,35 @@ WARNING: `username` and `password` are like input but {sechub} tries
          to hide these information in logs or at UI where possible.
          So do *NOT* use `input` for credentials!
 
+====== Example form based login with TOTP
+The TOTP configuration is generally never used without any other login mechanism. It is meant to be used as an additional authentication factor.
+In this example TOTP is used in combination with form based login, which is left out in the example for better readability.
+
+TIP: Most of the time it is enough to specify a `seed` for the TOTP configuration in SecHub.
```

Comment:
We have some variables to avoid naming failures etc.
Please use {sechub} variable here:

```suggestion
TIP: Most of the time it is enough to specify a `seed` for the TOTP configuration in {sechub}.
```